view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -294,7 +294,7 @@ define(
                         //This is required by magento to store the token
                         'is_active_payment_token_enabler' : this.storeCc,
                         'number_of_installments': this.installment(),
-                        'frontendType': 'default'
+                        'frontendType': 'luma'
```

Comment:
Does it have to be 'luma'? Luma is only a default theme for magento2, just like 'blank', so in most cases it won't be strictly 'luma' being used

Reply:
No, it doesn't but we wanted to synchronize frontendType throughout the project since we have both `luma` and `default` at different places, and we want to differentiate from other frontendTypes, 'hyva' for example. I decided on `luma` since it looked more informative but I have no preference, I can set them to `default` if that is more appropriate. Should I?

Reply:
Then I think default would be better indeed! If it doesn't cause issues of course